src/Traits/BranchDeployForgeInputs.php

```diff
@@ -31,7 +31,7 @@ protected function getDatabasePassword(): string {
 
     protected function getPhpVersion(): string {
         $version = strtolower($this->input->getOption('php-version'));
-        return in_array($version, ['php8.1', 'php8.2']) ? $version : 'php8.1';
+        return in_array($version, ['php8.1', 'php8.2']) ? $version : 'php8.2';
     }
```

Comment:
why do we need that?

Reply:
For creating a daemon with the correct version.

Reply:
the daemons are running in 8.1?

Reply:
Yeah, for new env I see as below
PHP version and daemon

<img width="1037" alt="Screenshot 2024-03-06 at 16 45 57" src="https://github.com/timberhubcom/laravel-deployments/assets/146819771/4e51a5a1-5b01-4713-9067-f08f3a54aeb9">
<img width="1075" alt="Screenshot 2024-03-06 at 16 46 17" src="https://github.com/timberhubcom/laravel-deployments/assets/146819771/f07de6ea-307a-48ef-80af-df25100072d4">

Reply:
I don't get it, why don't we move everything the 8.2?

Reply:
Because the next time we want to upgrade to 8.3 we need to have such a backward-compatibility (during testing)?

Reply:
ok good point